web/cypress/integration/route/create-route-with-upstream.spec.js

```diff
@@ -83,58 +81,53 @@ context('Create Route with Upstream', () => {
     cy.contains('Route').click();
 
     // edit existed router
-    cy.get('[title=Name]').type(routeName);
+    cy.get('[title=Name]').type(data.routeName);
     cy.contains('Search').click();
-    cy.wait(1000);
-    cy.contains(routeName).siblings().contains('Edit').click();
+    cy.contains(data.routeName).siblings().contains('Edit').click();
     cy.wait(1000);
     cy.contains('Next').click();
 
     // check if the changes have been saved
-    cy.get('#nodes_0_host').should('value', testData.testIp);
+    cy.get('#nodes_0_host').should('value', data.testIp);
 
     // select existed upstream_id will be disabled
     cy.get('#upstream_id').click();
-    cy.contains(upstreamName).click();
+    cy.contains(data.upstreamName).click();
     cy.get(domSelector.input).should('be.disabled');
-    cy.wait(1000);
 
     // select Custom upstream_id will not be disabled
     cy.contains('test_upstream').click();
     cy.contains('Custom').click();
     cy.get(domSelector.input).should('not.be.disabled');
 
     // change domain name/IP
-    cy.get('#nodes_0_host').clear().type(testData.verificationIp);
+    cy.get('#nodes_0_host').clear().type(data.verificationIp);
     cy.contains('Next').click();
     cy.contains('Next').click();
     cy.contains('Submit').click();
     cy.contains('Goto List').click();
     cy.url().should('contains', 'routes/list');
 
     // check if the changes have been saved
-    cy.get('[title=Name]').type(routeName);
+    cy.get('[title=Name]').type(data.routeName);
     cy.contains('Search').click();
-    cy.wait(1000);
-    cy.contains(routeName).siblings().contains('Edit').click();
-    cy.wait(1000);
+    cy.contains(data.routeName).siblings().contains('Edit').click();
     cy.contains('Next').click();
-    cy.get('#nodes_0_host').should('value', testData.verificationIp);
+    cy.get('#nodes_0_host').should('value', data.verificationIp);
   });
 
   it('should delete this test route and upstream', () => {
     cy.visit('/routes/list');
-    cy.get('[title=Name]').type(routeName);
+    cy.get('[title=Name]').type(data.routeName);
     cy.contains('Search').click();
-    cy.contains(routeName).siblings().contains('Delete').click();
+    cy.contains(data.routeName).siblings().contains('Delete').click();
     cy.contains('button', 'Confirm').click();
-    cy.get(testData.notification).should('contain', testData.deleteRouteSuccess);
+    cy.get(data.notification).should('contain', data.deleteRouteSuccess);
```

Comment:
no, data is different from domSelector.

data is used to track what we input or what we get, domSelector is a collection of dom selectors.